pygfunction/gfunction.py

```diff
@@ -259,6 +257,13 @@ def __init__(self, boreholes_or_network, alpha, time=None,
         # Check the validity of inputs
         self._check_inputs()
 
+        # needed to break circular imports
+        from .solvers import (
+            Detailed,
+            Equivalent,
+            Similarities
+        )
```

Comment:
There's some lingering circular dependencies among the files. Needed to import these here to break the circular import problems.